pkg/api/types.go

```diff
@@ -385,6 +388,9 @@ const (
 	// BetaStorageClassAnnotation represents the beta/previous StorageClass annotation.
 	// It's currently still used and will be held for backwards compatibility
 	BetaStorageClassAnnotation = "volume.beta.kubernetes.io/storage-class"
+	// AlphaStorageTopologyAnnotation is set in the StorageClass and gives the scheduler
+	// a hint that the volumes in this StorageClass have a topology constraint
+	AlphaStorageTopologyAnnotation = "storage.alpha.kubernetes.io/topology-key"
```

Comment:
The value can be any label that exists on the PV + Node.  I can add that to the comment.  

The pod affinity feature specifies label selector and topologyKey as separate fields.  Maybe we can do the same here.
{{LabelSelector: P1, TopologyKey: "node"}}